.github/workflows/ci-lambda.yml

```diff
@@ -17,28 +17,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        projects: 
+        projects:
           [
-            { 
-              projectName: "sftp-outbound-transfer-lambda", 
-              projectRoot: "ops/terraform/services/eft/lambda_src/sftp_outbound_transfer", 
-              testsRoot: "ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/tests", 
-              pythonVersion: "3.12" 
+            {
+              projectName: "sftp-outbound-transfer-lambda",
+              projectRoot: "ops/terraform/services/eft/lambda_src/sftp_outbound_transfer",
+              testsRoot: "ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/tests",
+              pythonVersion: "3.12",
             },
-            { 
-              projectName: "update-pipeline-slis-lambda", 
-              projectRoot: "ops/terraform/services/pipeline/modules/bfd_pipeline_slis/lambda_src", 
-              testsRoot: "ops/terraform/services/pipeline/modules/bfd_pipeline_slis/lambda_src/tests", 
-              pythonVersion: "3.11" 
-            }
```

Comment:
This is the only relevant change to this file as this Lambda no longer exists and thus needs to be removed.

All other changes are formatting.